modular_splurt/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -194,7 +194,7 @@
 	color = "#1a5fa1"
 	quality = DRINK_NICE
 	taste_description = "blue orange"
-	glass_icon = 'modular_splurt/icons/obj/drinks.dmi'
+	glass_icon = 'icons/obj/drinks.dmi'
```

Comment:
You actually don't need to set the glass_icon if the icon isn't different from the nonmodular drinks.dmi. The variable is only in case the icon for the glass will be in a different file

Reply:
Thank you for pointing out this minor oversight. The changes have been applied in a follow-up commit.